src/Coduo/PHPHumanizer/Number/Ordinal/StrategyInterface.php

```diff
@@ -4,10 +4,15 @@
 
 interface StrategyInterface
 {
+	/** 
+	 * @return boolean
+	 */
+	public function isPrefix();
+
     /**
      * @param int|float $number
      *
      * @return string
      */
-    public function ordinalSuffix($number);
+    public function ordinalIndicator($number);
```

Comment:
Some languages has ordinal as prefix, e.g Indonesian, Malay, Chinese, ...
Ex: 
"1st" in English = "ke-1" in Indonesian